### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by chrisjsewell**
Change `✏️ REWRITE` to `👀 REVISE`
Not sure what the best emoji+keyword is for this? but its basically for minor things that should not show up directly in a changelog, like typos or very minor code changes (i.e. nothing that users/developers would really care about) 
I would call it `NITPICK`, but people who have made PRs might not appreciate this wording for their commit lol

e.g. I would have used this for
https://github.com/executablebooks/sphinx-book-theme/pull/150#issuecomment-669979005 and
https://github.com/executablebooks/sphinx-book-theme/pull/155

### CODE REVIEW

**Comment by choldgraf** [comment]
lol or we could have an undocumented PR tag for when we want to be passive aggressive: 🚲 BIKESHED:

but seriously - "REVISE" is fine with me. In other projects we have just used a generic `🔧 MAINT:` that lumps in "REVISE" and "REFACTOR" under general maintenance category. But if we wanna keep them separate, I'm fine w/ revise.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -325,12 +325,12 @@ There is no need to mention you also added unit tests when adding a new feature.
...
 
 - `‼️ BREAKING:` (`:bangbang:`) — to introduce a back-incompatible change(s) (and/or remove deprecated code).
-- `✨  NEW:` (`:sparkles:`) — to introduce a new feature(s).
+- `✨ NEW:` (`:sparkles:`) — to introduce a new feature(s).
 - `👌 IMPROVE:` (`:ok_hand:`) — to improve an existing code/feature (with no breaking changes).
 - `🐛 FIX:` (`:bug:`) — to fix a code bug.
 - `📚 DOCS:` (`:books:`) — to add new documentation.
-- `✏️ REWRITE:` (`:pencil2:`) — to apply fixes (like correcting typos) to existing documentation.
-- `🧪  TEST:` (`:testube:`) — to add additional testing only.
+- `👀 REVISE:` (`:eyes:`) — to revise minor issues (like typos) which should not appear in a changelog.
```
Comment (chrisjsewell) [inline CONTRIBUTING.md]:
```suggestion
- `🔧 MAINTAIN:` (`:wrench:`) — to make minor changes (like fixing typos) which should not appear in a changelog.
```

**Comment by choldgraf** [comment]
I love this fish

**Comment by pradyunsg** [comment]
> NITPICK, but people who have made PRs might not appreciate this wording for their commit lol

😂

FWIW, I certainly would've called it that!

**Pull request merged**